common/protobuf/Control.proto

```diff
@@ -0,0 +1,33 @@
+package Packet;
+
+message Control
+{
+    enum ShootMode
+    {
+        KICK = 1;
+        CHIP = 2;
+    }
+
+    enum TriggerMode
+    {
+        IMMEDIATE = 1;
+        ON_BREAK_BEAM = 2;
+        STAND_DOWN = 3;
+    }
+
+    enum Song
+    {
+        STOP = 1;
+        FIGHT_SONG = 2;
+    }
+
+    required float xVelocity = 1         [default = 0];
+    required float yVelocity = 2         [default = 0];
+    required float aVelocity = 3         [default = 0];
+    required float dVelocity = 4         [default = 0];
+    required uint32 kcStrength = 5       [default = 255];
+    required ShootMode shootMode = 6     [default = KICK];
+    required TriggerMode triggerMode = 7 [default = STAND_DOWN];
+    required bool firmwareEnable = 8     [default = false];
```

Comment:
Gotcha.  Do we want that in the control packet or can the robot infer that when it receives the first firmware packet?

Also, it would be best to keep this PR to just the control packet refactor, then have another PR later that contains the firmware update stuff.  It may be a while until the firmware stuff gets done.